src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -161,177 +132,127 @@ class WorkspaceInvitePage extends React.Component {
-                                    onSelectRow={this.toggleOption}
-                                    onChangeText={this.updateOptionsWithSearchTerm}
-                                    onConfirmSelection={this.inviteUser}
-                                    headerMessage={headerMessage}
-                                    hideSectionHeaders
-                                    boldStyle
-                                    shouldDelayFocus
-                                    shouldFocusOnSelectRow={!Browser.isMobile()}
-                                    textInputLabel={this.props.translate('optionsSelector.nameEmailOrPhoneNumber')}
-                                />
-                            </View>
-                            <View style={[styles.flexShrink0]}>
-                                <FormAlertWithSubmitButton
-                                    isDisabled={!this.state.selectedOptions.length}
-                                    isAlertVisible={this.getShouldShowAlertPrompt()}
-                                    buttonText={this.props.translate('common.next')}
-                                    onSubmit={this.inviteUser}
-                                    message={this.props.policy.alertMessage}
-                                    containerStyles={[styles.flexReset, styles.flexGrow0, styles.flexShrink0, styles.flexBasisAuto, styles.mb5]}
-                                    enabledWhenOffline
-                                    disablePressOnEnter
-                                />
-                            </View>
-                        </FullPageNotFoundView>
-                    );
-                }}
-            </ScreenWrapper>
-        );
-    }
+                        </View>
+                        <View style={[styles.flexShrink0]}>
+                            <FormAlertWithSubmitButton
+                                isDisabled={!selectedOptions.length}
+                                isAlertVisible={shouldShowAlertPrompt}
+                                buttonText={props.translate('common.next')}
+                                onSubmit={inviteUser}
+                                message={props.policy.alertMessage}
+                                containerStyles={[styles.flexReset, styles.flexGrow0, styles.flexShrink0, styles.flexBasisAuto, styles.mb5]}
+                                enabledWhenOffline
+                                disablePressOnEnter
+                            />
+                        </View>
+                    </FullPageNotFoundView>
+                );
+            }}
+        </ScreenWrapper>
+    );
 }
 
 WorkspaceInvitePage.propTypes = propTypes;
 WorkspaceInvitePage.defaultProps = defaultProps;
+WorkspaceInvitePage.displayName = 'WorkspaceInvitePage';
 
 export default compose(
     withLocalize,
     withPolicyAndFullscreenLoading,
-    withNetwork(),
```

Comment:
Please remove `withLocalize` and  use `useLocalize`  hook.

Reply:
@Santhosh-Sellavel

We can't use `useCallback` here.
We need to call the function on mount and on network connect only. We can't do this using `useCallback`

Reply:
We should update on props.policyMembers, props.personalDetails changes too

Reply:
Please check the old code. It doesn't work that way

Reply:
NVM